site/docs/v1/tech/apis/_application-response-body.adoc

```diff
@@ -98,7 +98,7 @@ Indicates if a Refresh Token should be issued from the Login API.
 Indicates if the Login API should require an API key. If you set this value to `false` and your FusionAuth API is on a public network, anyone may attempt to use the Login API.
 
 [field]#application.oauthConfiguration.authorizedOriginURLs# [type]#[Array<String>]#::
-An array of URLs that are the authorized origins for FusionAuth OAuth.
+An array of URLs that are the authorized origins for FusionAuth OAuth. If configured, these URLs will be allowed to embed a FusionAuth hosted login page, otherwise the `X-Frame-Options` header will be set to `DENY`.
```

Comment:
Suggestion: 

>
> One or more HTTP origins that are allowed to use the browser based grants, and allowed to load the login pages in an Iframe. 
> {empty line}
>  When this configuration is omitted, all HTTP origins are allowed to use the browser based grants and the HTTP response header of `X-Frame-Options: DENY` will be added to each response to completely restrict the use of Iframes. 

We may also want to add a short comment on support of this strategy. It should work, but it is not a tested use case that we document anywhere else.